js/src/utils/saveController.js

```diff
@@ -127,6 +169,21 @@
         _this.set("windowObjects", windowObjects, {parent: "currentConfig"} );   
       });
 
+      jQuery.subscribe('ANNOTATIONS_LIST_UPDATED', function(event, options) {
+        if (!_this.windowsAnnotationsLists) {
+          _this.windowsAnnotationsLists = {};
+        }
+        _this.windowsAnnotationsLists[options.windowId] = options.annotationsList;
+        jQuery.publish('annotationListLoaded.' + options.windowId);
+      });
+
+      jQuery.subscribe('WINDOW_ELEMENT_UPDATED', function(event, options) {
+        if (!_this.windowsElements) {
+          _this.windowsElements = {};
+        }
+        _this.windowsElements[options.windowId] = options.element;
+      });
+
```

Comment:
Some items are not saved to localStorage because they take up a lot of space, and are only needed during runtime.  This information will be regenerated from a localStorage config, so there is no need to include it in that.